README.md

```diff
@@ -31,29 +31,44 @@ The eZ Publish Proprietary License gives you the right to use the source code
 for making your own commercial software. It allows you full protection of your
 work made with eZ Publish. You may re-brand, license and close your source
 code. eZ Publish is not free of charge when used under the terms of the
-Professional License. For pricing and ordering, please contact us at
-info@ez.no or visit http://ez.no/products/proprietary_license_options
+Professional License. For pricing and ordering, please contact info@ez.no
+or visit http://ez.no
 
 
-eZ Publish features
--------------------
+Main eZ Publish features
+------------------------
 - User defined content classes and objects
-- Advanced search engine
-- Role based permissions system
-- Advanced template engine
 - Version control
-- Professional workflow management
-- Multi-lingual support
-- Support for Unicode
-- Task system for easy collaboration
-- Image conversion and scaling
+- Advanced multi-lingual support
+- Built in search engine
+- Separation of content and presentation layer
+- Fine grained role based permissions system
+- Content approval and scheduled publication
+- Multi-site support
+- Multimedia support with automatic image conversion and scaling
+- RSS feeds
+- Contact forms
+- Built in webshop
+- Flexible workflow management system
+- Full support for Unicode
+- Template engine
+- A REST API
 - Database abstraction layer
+- MVC architecture
```

Comment:
Well, it definitely is. It uses kinda funny naming and breaks the philosophy with templates, but you got 2-level controllers, the model (ezco and friends) and the presentation(templates)